src/ui/components/CommandPalette/CommandButton.tsx

```diff
@@ -4,22 +4,28 @@ import { connect, ConnectedProps } from "react-redux";
 import { actions } from "ui/actions";
 import { Command } from "./CommandPalette";
 
-type CommandButtonProps = PropsFromRedux & { command: Command };
+type CommandButtonProps = PropsFromRedux & { command: Command; selected: boolean };
```

Comment:
Usually, this is more appropriately named as `active`.
(sorry if this way too nitpicky)

Reply:
Nope - I like it